ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/FastSyncDownloader.java

```diff
@@ -80,7 +80,7 @@ public CompletableFuture<FastSyncState> start() {
     if (!running.compareAndSet(false, true)) {
       throw new IllegalStateException("SyncDownloader already running");
     }
-    LOG.info("Starting sync");
+    LOG.info("Starting fast sync");
```

Comment:
it's not clean but this class is also used by snapsync. I think it's why we removed the fast